app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt

```diff
@@ -57,7 +58,7 @@ class BluetoothManagerImpl(
                         foregroundNotification()
                     )
                 )
-                start()
+                if (BluetoothAdapter.getDefaultAdapter().bluetoothLeAdvertiser != null) start()
```

Comment:
Seems like `bluetoothLeAdvertiser` needs both location permission, and bluetooth turned on. In my PR I handle bluetooth, but I don't ask for location permission (I ask for location permission in the first launch of the app)

Anyway, this check fixes the crash. I'll create another PR that will ask for location permission in home screen.